bin/jobsub_cmd

```diff
@@ -103,8 +132,62 @@ def main() -> None:
 
     # combine jobsub_q as well
     if cmd == "jobsub_q":
+
         execargs.insert(0, "-global")
 
+        if default_constraint:
+            execargs.extend(
+                [
+                    "-constraint",
+                    f'''Jobsub_Group=="{os.environ['GROUP']}"''',
+                ]
+            )
+
+        if default_formatting:
```

Comment:
Do we want to actually default to the old jobsub format?  Or keep  the current default format, with the ability to get the old format?  That was my understanding from yesterday's meeting (I'm compiling those minutes now...sorry for the delay).  @retzkek  - how did you interpret this decision?

Reply:
Yes, I was going to bring this up for discussion again; Should we not default to the old behavior? If people are reading current jobsub output, say in a script, then we should keep the old output... do we think users are wrapping jobsub_q or parsing its output in other scripts?

Reply:
Resolved - jobsub working group decided to have default behavior match old jobsub.  So this is good.